engine/src/main/java/org/terasology/engine/world/block/family/BlockFamilyLibrary.java

```diff
@@ -37,7 +37,7 @@ public BlockFamilyLibrary(ModuleEnvironment moduleEnvironment, Context context)
         for (Class<?> entry : moduleEnvironment.getTypesAnnotatedWith(RegisterBlockFamily.class)) {
 
             if (!BlockFamily.class.isAssignableFrom(entry)) {
-                logger.error("Cannot load {}, must be a subclass of BlockFamily", entry.getSimpleName());
+                logger.atError().addArgument(() -> entry.getSimpleName()).log("Cannot load {}, must be a subclass of BlockFamily");
```

Comment:
Only evaluated in unexpected cases during construction of `BlockFamilyLibrary` so performance impact acceptable.
```suggestion
                logger.error("Cannot load {}, must be a subclass of BlockFamily", entry.getSimpleName()); //NOPMD
```